config/bin_version_strings.cfg

```diff
@@ -42,6 +42,7 @@ comgt::"comgt\ version\ [0-9]\.[0-9]+"
 conntrack::"conntrack\ v[0-9]\.[0-9]\.[0-9]+\ \(conntrack-tools\)"
 conntrack::"conntrack\ v[0-9]\.[0-9]\.[0-9]\ \(conntrack-tools\)"
 coreutils::"\(GNU\ coreutils\)\ [0-9]\.[0-9]+$"
+chrony::"chrony[cd]\ \(chrony\)\ version [0-9]\.[0-9]"
```

Comment:
could we do something like this?
chrony::"**^**chrony[cd]\ \(chrony\)\ version [0-9]\.[0-9]**$**"

Same on the other version identifiers